.aspect/workflows/config.yaml

```diff
@@ -8,6 +8,9 @@ tasks:
   - gazelle:
   - configure:
   - test:
+      hooks:
```

Comment:
Want to add comments in code for the various flags, since this is an example?

Reply:
Good call. Updated.